common/actions/swapTypes.js

```diff
@@ -0,0 +1,66 @@
+import {
+  SWAP_DESTINATION_AMOUNT,
+  SWAP_DESTINATION_KIND,
+  SWAP_ORIGIN_AMOUNT,
+  SWAP_ORIGIN_KIND,
+  SWAP_UPDATE_BITY_RATES,
+  SWAP_DESTINATION_ADDRESS,
+  SWAP_RESTART,
+  SWAP_LOAD_BITY_RATES,
+  SWAP_STOP_LOAD_BITY_RATES,
+  SWAP_STEP,
+  SWAP_REFERENCE_NUMBER
+} from './swapConstants';
+
+export type Pairs = {
+  ETHBTC: number,
+  ETHREP: number,
+  BTCETH: number,
+  BTCREP: number
+};
+
+export type ReferenceNumberSwapAction = {
+  type: SWAP_REFERENCE_NUMBER,
+  value: string
+};
+export type OriginKindSwapAction = {
+  type: SWAP_ORIGIN_KIND,
+  value: string
+};
+export type DestinationKindSwapAction = {
+  type: SWAP_DESTINATION_KIND,
+  value: string
+};
+export type OriginAmountSwapAction = {
+  type: SWAP_ORIGIN_AMOUNT,
+  value: string | number
```

Comment:
better to have one type and cast in action creator or calling code (if it is indeed a number as a string if its something else - let me know)